src/rgw/rgw_rados.h

```diff
@@ -2525,6 +2531,10 @@ class RGWRados : public AdminSocketHook
       const auto& zonegroups_by_api = current_period.get_map().zonegroups_by_api;
       if (zonegroups_by_api.find(api) != zonegroups_by_api.end())
         return true;
+    } else if ( zonegroup.is_default_created ) {
+      if (zonegroup.api_name == api) {
```

Comment:
do you think the `is_default_created` part is really neccessary? i'm guessing it would work fine if it was just `else if (zonegroup.api_name == api)` here